nevergrad/functions/photonics/core.py

```diff
@@ -21,10 +21,10 @@
 
 import numpy as np
 from nevergrad.parametrization import parameter as p
-from . import photonics
-from .. import base
-
-
+#from . import photonics
+import photonics
+#from .. import base
+from . import base
```

Comment:
```suggestion
from . import photonics
from .. import base
```
These modifications would not work for the package.
If you want to access the file you are modifying from an external script, you should do
`from nevergrad.functions.photonics import photonics`
This will work either from wherever you want if nevergrad is installed (I can guide you through it, since you are modifying the package there is a specific way to install it without `pip` making some kind of a "frozen" copy). or from the nevergrad repo directory (i.e the one containing `setup.py`, a `nevergrad` directory, the license, etc

Reply:
OK, thank you ! Actually, I tried to check only the methods in photonics.py, in order to understand why _a_ and _e_ don't have the same size in the _solar()_ function. I am not yet familiar with pytest or mypy, I am a bit lost with the quantity of errors those commands return me, even in files I did not modify at all... so I tried another way : I just run the file test_core.py and receive the error "attempted relative import with no known parent package". 
I began to follow all the documentation in Nevergrad to understand how to perform an optimization directly with Nevergrad (for now, I used my DE and not Nevergrad), and how to test differents methods/packages/things in it. I will replace the _import_ by the original ones, copy all the files in a local space, modify and check all from it, and maybe when the code will be clearer I will update this PR !

Reply:
Update : the pytest is working, but now the "commit button" is broken... I am working on it !

Reply:
don't worry about the commit button for now, this can be fixed easily and is not very important. I may be able to do it for you (I'm still unsure it would work since I cant commit to your branch, but I can try).

For pytest, indeed `pytest nevergrad` runs aaaaaalll the tests in nevergrad. but you can select a subsample to run. In you case `pytest nevergrad/functions/photonics` should be enough, and muuch faster ;)